lib/sqlalchemy/orm/mapped_collection.py

```diff
@@ -38,28 +49,35 @@ class _PlainColumnGetter:
 
     __slots__ = ("cols", "composite")
 
-    def __init__(self, cols):
+    def __init__(self, cols: List[KeyedColumnElement[Optional[_KT]]]) -> None:
         self.cols = cols
         self.composite = len(cols) > 1
 
-    def __reduce__(self):
+    def __reduce__(
+        self,
+    ) -> Tuple[
+        Type[_SerializableColumnGetterV2],
+        Tuple[List[Tuple[str, Optional[_KT]]]],
+    ]:
         return _SerializableColumnGetterV2._reduce_from_cols(self.cols)
 
-    def _cols(self, mapper):
+    def _cols(
+        self, mapper: Mapper[_KT]
+    ) -> List[KeyedColumnElement[Optional[_KT]]]:
         return self.cols
 
-    def __call__(self, value):
+    def __call__(self, value: _KT) -> Union[_KT, Tuple[_KT, ...]]:
         state = base.instance_state(value)
         m = base._state_mapper(state)
 
-        key = [
+        key = [  # type: ignore
             m._get_state_attr_by_column(state, state.dict, col)
             for col in self._cols(m)
         ]
         if self.composite:
             return tuple(key)
         else:
-            return key[0]
+            return key[0]  # type: ignore
```

Comment:
`KeyedColumnElement` is actually from `sql.elements` and its `key` attribute is of `str` type - so it might be that the elements in `key` list here are strings. We probably can't do much more here, since `_get_state_attr_by_column()` has Mike's annotations - just wanted to add a bit of context from `sql.elements`.